### PR TITLE
Fixed several typos

### PR DESCRIPTION
**Pull request by bitdaw**
I found and fixed a few typos 🙂

### CODE REVIEW

File: Content/Part 3/14-a-new-logo.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ More designs were posted to the group:
 *Further iterations on the logo design from Jason Santa Maria.*
 
 The members of the mailing list didn’t seem to agree on WordPress’ aesthetic. On one hand, there were people who felt that the logo should represent warmth and community, and on the other hand, something classic and elegant. To reach a consensus, discussions happened offline. Khaled <a href="http://lists.wordpress.org/pipermail/wp-design/2005-April/000192.html">reported back</a>:
-<blockquote>WordPress is meant to be the Jaguar or Aston Martin of Blogging tools. [...] that line sets the stage for what the design of the branding should be. Elegance, polished, and impecably [sic] designed is where we should be aiming.</blockquote>
+<blockquote>WordPress is meant to be the Jaguar or Aston Martin of Blogging tools. [...] that line sets the stage for what the design of the branding should be. Elegance, polished, and impeccably [sic] designed is where we should be aiming.</blockquote>
```
Comment (coffee2code) [inline Content/Part 3/14-a-new-logo.md]: This is a case where the text in question is a direct quote copied from its source. The use of "[[sic](https://www.merriam-webster.com/dictionary/sic)]" was added in the book's copy to indicate that the misspelling is known but being left as-is to mirror the original.

File: Content/Part 3/15-wordpress-inc.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Even as the party was going on, trouble was brewing. In February, a <a href="htt
...
+<blockquote>The content in /articles is essentially advertising by a third party that we host for a flat fee. I'm not sure if we're going to continue it much longer, but we're committed to this month at least, it was basically an experiment. However around the beginning of February [sic] donations were going down as expenses were ramping up, so it seemed like a good way to cover everything. The adsense on those pages is not ours and I have no idea what they get on it, we just get a flat fee. The money is used just like donations but more specifically it's been going to the business/trademark expenses so it's not entirely out of my pocket anymore.</blockquote>
```
Comment (coffee2code) [inline Content/Part 3/15-wordpress-inc.md]: This is another case where the text in question is a direct quote copied from its source. The use of "[[sic](https://www.merriam-webster.com/dictionary/sic)]" was added in the book's copy to indicate that the misspelling is known but being left as-is to mirror the original.

File: Content/Part 3/14-a-new-logo.md (hunk 1 above)
Comment (bitdaw) [reply Content/Part 3/14-a-new-logo.md]: Understood, wasn't aware of the meaning of [sic], thanks for explaining!

**Comment by coffee2code** [comment]
Hi @AndrewDawes! Thanks for the PR. Everything looks great except that you fixed two typos that were known and left as-is due to being present in the originally quoted text. I've commented inline.

**Comment by bitdaw** [comment]
Hi @coffee2code, thanks for the explanation, wasn't aware of the [sic] thing. Whats the best way for me to remove those 2 files from my PR without making things messy here with extra commits? 🙂

**Comment by coffee2code** [comment]
The easiest way would be to just make another commit to the branch you created the PR from and I'll just squash all of the PR commits into one.

Or you should be able to update that same branch and `git commit --amend` to update the existing commit with your changes, then `git push --force` to force your previous commit to be overwritten. (Which understandably can be a lot depending on your git experience.)

**Comment by bitdaw** [comment]
@coffee2code Thanks for explaining, I think I did it right using `git checkout [commit-ref] -- [filename]` and then `git commit --amend`. Thanks a lot for the help, learned something new 😄

**Comment by coffee2code** [comment]
@AndrewDawes Your update was perfect. Thanks again for the PR!

**Pull request merged**